composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "jojoe77777/formapi",
+    "description": "Simple API for creating forms on PocketMine-MP.",
+    "type": "library",
+    "license": "LGPL-3.0",
+    "require": {
+        "pocketmine/pocketmine-mp": "^4.0.0"
```

Comment:
5.0.0 now also

Reply:
Does Composer support using 2 major versions at the same time?

Reply:
`^4.0.0 || ^5.0.0`